tox.ini

```diff
@@ -45,7 +45,7 @@ deps =
 allowlist_externals =
     /bin/bash
 basepython = python3.8
-commands = /bin/bash -c '{envpython} -m pytest -v --profile databricks_sql_endpoint {posargs} -n4 tests/functional/adapter/*'
+commands = /bin/bash -c '{envpython} -m pytest -v --profile databricks_sql_endpoint {posargs}  -n4 tests/functional/adapter/*'
```

Comment:
nit: is this a extra space?

Reply:
Assuming that the extra space was not intended to be merged, I removed it here:
https://github.com/dbt-labs/dbt-spark/pull/582/files

If the space was intentional, we can add it back if it makes sense.